index.js

```diff
@@ -18,7 +18,8 @@ module.exports = function (obj, opts) {
 		return function (a, b) {
 			return cmpOpt(
 				{ key: a, value: node[a] },
-				{ key: b, value: node[b] }
+				{ key: b, value: node[b] },
+				{ get: function (k) { return node[k]; } }
```

Comment:
might be nice to do some optimization here:
```suggestion
				cmpOpt.length > 2 ? { __proto__: null, get: function (k) { return node[k]; } } : void undefined
```

so we’re not creating the object unless the function expects it (also, make it a null object for safety)

Reply:
Looks good - please commit.